### ISSUE TITLE
`DJ013`: False positive for multiple `@receiver` decorators

### ISSUE THREAD

**Issue by ngnpope** [charliermarsh/ruff#2850]
For the following:

```python
from django.db import models
from django.db.models.signals import post_save
from django.dispatch import receiver

class ModelA(models.Model):
    pass

class ModelB(models.Model):
    pass

@receiver(post_save, sender=ModelA)
@receiver(post_save, sender=ModelB)
def on_post_save(**kwargs):
    pass
```

I get the following:

```console
❯ ruff --version
ruff 0.0.246

❯ ruff --isolated --select DJ013 bug.py
bug.py:12:2: DJ013 `@receiver` decorator must be on top of all the other decorators
Found 1 error.
```

I expect no errors. When fixing, a test should also be added for multiple `@receiver` decorators with other decorators too to ensure all `@receiver` decorators are the outermost.
